group_layers/group-layers/src/main/java/com/esri/samples/group_layers/GroupLayersSample.java

```diff
@@ -78,40 +82,63 @@ public void start(Stage stage) {
-      ArcGISSceneLayer devATrees = new ArcGISSceneLayer("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_Trees/SceneServer");
-      FeatureLayer devAPathways = new FeatureLayer(new ServiceFeatureTable(" https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_Pathways/FeatureServer/1"));
-      FeatureLayer devProjectArea = new FeatureLayer(new ServiceFeatureTable("https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevelopmentProjectArea/FeatureServer/0"));
-
-      // create a group layer from scratch by adding the layers as children
-      groupLayer = new GroupLayer();
-      groupLayer.setName("Group: Dev A");
-      groupLayer.getLayers().addAll(Arrays.asList(devATrees, devAPathways, devABuildings));
-
-      // add the group layer and other layers to the scene as operational layers
-      scene.getOperationalLayers().addAll(Arrays.asList(groupLayer, devBBuildings, devProjectArea));
-
-      // zoom to the extent of the group layer when the child layers are loaded
-      groupLayer.getLayers().forEach(childLayer ->
-        childLayer.addDoneLoadingListener(() -> {
-          if (childLayer.getLoadStatus() == LoadStatus.LOADED) {
-            sceneView.setViewpointCamera(new Camera(groupLayer.getFullExtent().getCenter(), 700, 0, 60, 0));
+      // create two group layers, set the visibility modes and add layers
+      projectAreaGroupLayer = new GroupLayer();
+      projectAreaGroupLayer.setName("Project area group");
+      projectAreaGroupLayer.setVisibilityMode(GroupVisibilityMode.INDEPENDENT);
+      projectAreaGroupLayer.getLayers().addAll(Arrays.asList(
+        (new FeatureLayer(new ServiceFeatureTable("https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevelopmentProjectArea/FeatureServer/0"))),
+        (new FeatureLayer(new ServiceFeatureTable("https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_Pathways/FeatureServer/1"))),
+        (new ArcGISSceneLayer("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_Trees/SceneServer"))
+      ));
+
+      buildingsGroupLayer = new GroupLayer();
+      buildingsGroupLayer.setName("Buildings group");
+      buildingsGroupLayer.setVisibilityMode(GroupVisibilityMode.EXCLUSIVE);
+      buildingsGroupLayer.getLayers().addAll(Arrays.asList(
+        (new ArcGISSceneLayer("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevA_BuildingShells/SceneServer")),
+        (new ArcGISSceneLayer("https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/DevB_BuildingShells/SceneServer"))
+      ));
+
+      // load the group layers
+      projectAreaGroupLayer.loadAsync();
+      buildingsGroupLayer.loadAsync();
+
+      // Display an alert if the child layers in the group layers fail to load
+      projectAreaGroupLayer.getLayers().forEach(layer ->
```

Comment:
I think it's more readable to name each layer childLayer as we had before: 

```suggestion
      projectAreaGroupLayer.getLayers().forEach(childLayer ->
```